FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -66,12 +73,26 @@ @implementation FPRAppActivityTracker
 + (void)load {
   // This is an approximation of the app start time.
   appStartTime = [NSDate date];
+
+  if ([[[NSBundle mainBundle] objectForInfoDictionaryKey:@"fireperf_disable_dd"] boolValue] == NO) {
```

Comment:
minor nit: Add comments to qualify what are we doing below and why? This is a critical piece of code for someone who wants take a look at the code.